### ISSUE TITLE
Dark Theme

### ISSUE THREAD

**Issue by wayneconnolly** [coderaiser/cloudcmd#332]
It would be nice to have theming or at least a dark theme.

**Comment by coderaiser** [comment]
Take a look here https://github.com/cloudcmd/theme

**Comment by kingfenix** [comment]
how would we apply a theme when running the container via docker in UNRAID?

**Comment by coderaiser** [comment]
> how would we apply a theme when running the container via docker in UNRAID?

There is no such a feature, but good idea for a PR :). What is `UNRAID`?

**Comment by kingfenix** [comment]
its a NAS OS with a very large community.

https://unraid.net/

**Comment by carmatana** [comment]
Hello,

New to cloud commander, very impressed so far !!

thanks 

I running it in docker

Checking if the dark mode can be implemented in this setting 

thanks for the attention

**Comment by coderaiser** [comment]
Yep, are you up for a PR?

**Comment by lpmrfentazis** [comment]
It would be great if themes could be changed in the settings, ideally for each client separately

**Comment by carmatana** [comment]
> Yep, are you up for a PR?

Could you explain how to implement dark theme in docker ?

**Comment by coderaiser** [comment]
I can explain but this goes little bit beyond scope of this repo. You can support the project with some kind of donation so I can 
- hire designer, since dark them it is not just black color, or provide styles
according to examples from https://github.com/cloudcmd/theme

- or consult you about docker layers https://docs.docker.com/build/guide/layers/

**Comment by coderaiser** [comment]
Just landed dark theme in [v17.3.0](https://github.com/coderaiser/cloudcmd/releases/tag/v17.3.0) 🎉 

Is it works for you?

**Comment by DieMetRik** [comment]
Work only after restart cloud commander
